bin/generator/utils.dart

```diff
@@ -4,38 +4,69 @@ import 'package:recase/recase.dart';
 extension DartStyleName on String {
   /// Dart property reserved keywords
   static final Iterable<String> _reservedKeywords = [
-    'in',
-    'for',
-    'const',
-    'final',
+    'abstract',
+    'as',
+    'assert',
+    'async',
+    'await',
+    'break',
+    'case',
+    'catch',
     'class',
-    'call',
-    'if',
+    'const',
+    'continue',
+    'covariant',
+    'default',
+    'deferred',
+    'do',
+    'dynamic',
     'else',
     'enum',
-    'assert',
-    'super',
+    'export',
     'extends',
+    'extension',
+    'external',
+    'factory',
+    'false',
+    'final',
+    'finally',
+    'for',
+    'Function',
+    'get',
+    'hide',
+    'if',
+    'implements',
+    'import',
+    'in',
+    'interface',
     'is',
+    'late',
+    'library',
+    'mixin',
+    'new',
+    'null',
+    'on',
```

Comment:
Are some allowed short words also prefixed with `$`?
This is just an example, because I tried all of Dart's reserved keywords while compiling this list. The list here is completely forbidden in Dart.